css/style.css

```diff
@@ -1,10 +1,142 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
+ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+header {
+  position: fixed;
+  top: 0;
+  min-height: 75px;
+  padding: 0px 50px;
+  display: flex;
+  justify-content: flex;
+  align-items: center;
+  background-color: #ffffff;
+ 
+}
+@media (max-width: 500px) {
+  header {
+    flex-wrap: wrap;
+  }
+}
+
+  
+.logo {
+    margin-top: 15px;
```

Comment:
This rule is forcing the logo down so it is not centered vertically in the nav bar.